src/tests/simulation/tests/integration_tests/scale/cascade_materialized_view.rs

```diff
@@ -32,6 +32,7 @@ const MV5: &str = "create materialized view m5 as select * from m4;";
 async fn test_simple_cascade_materialized_view() -> Result<()> {
     let mut cluster = Cluster::start(Configuration::for_scale()).await?;
     let mut session = cluster.start_session();
+    let arrangement_backfill_is_enabled = session.is_arrangement_backfill_enabled().await?;
```

Comment:
Why could this vary during runtime? Didn't quite get it. 😕

Reply:
I assume it's for the tests to pass when the default value for `enable_arrangment_backfill` is set to true **manually**.

Can we integrate the tests with arrangement backfill into CI?

Reply:
> I assume it's for the tests to pass when the default value for enable_arrangment_backfill is set to true manually.

> Can we integrate the tests with arrangement backfill into CI?

Yeah that's correct. Will do in separate PR, requires more refactor.